### ISSUE TITLE
Cannot create maven package (opentsdb-protobuf) for OpenTSDB 3.0 due to javax.annotation

### ISSUE THREAD

**Issue by peteisace** [OpenTSDB/opentsdb#2097]
Steps:

1)
git checkout 3.0
maven package -Pdocker

Error: 
src/main/java/net/opentsdb/utils/Parsing.java
Unknown symbol: DoubleConsts
Unknown symbol: FDBigInteger

So I see that the package sun.misc has been moved to jdk.internal.math and replace the imports.

2) Tests fail now, so I run it without the tests. Not ideal
3) maven package -Pdocker -Dmaven.ignore.test=true

Error:
cannot find symbol: class Generated
location: package javax.annotation

I've tried

3.1) Changing the import to javax.annotation.processing.Generated, but of course this doesn't help; it's a generated file, so it is overwritten once I try to compile again.
3.2) Adding javax.annotation (1.3.2) into the pom.xml at the root. No change.
3.3) Adding javax.annotation (1.3.2) into the pom.xml at ~/implentation/protobuf/pom.xml

And now I'm stuck.

This seems to be an extra-ordinary work to get this to run; am I running this with incorrect settings? JDK is 14, maven 3.6.3. Haven't done anything to the defaults, not running multiple JDK's or anything, not set any environment variables.

Advice would be appreciated.

**Comment by manolama** [comment]
Yup, that's cause we're still stuck on JDK 1.8. It won't compile with 9+ since I added those low level number parsing routines. I'll target 11 shortly for 3.0 and I have to get my production environment setup with 11 in the next couple of weeks.

**Comment by manolama** [comment]
Well goody. I totally goofed up the 3.0 layout so that it's now much harder to move to 9+ modules. This will take a fair bit of work. If anyone's gone through the migration for similar projects in the past, ping me with your tips and suggestions. Thanks.

**Comment by MarkGetteoh** [comment]
Is this issue solved , i got the same, any idea provided here or other site like STACK OVERFLOW , cannot find useful point on google
